src/cf-core/usage.md

```diff
@@ -496,10 +529,6 @@ To create a 4:3 flexible video player, add the `__4_3` modifier to the container
 </div>
```

Comment:
I can't remember if this was a Github preview or Atom preview. I'm okay merging with this not resolved, because it should be easier to see once we generate the new docs.